### PR TITLE
#130 - Share multiple files in one go

### LINKED ISSUE #130: Share multiple files in one go
- In`View Files` option, users can select multiple files by clicking on the checkbox
- Add a share option at top that allows user to share multiple files in one go

### PR DESCRIPTION
**Pull request by anandkumarkparmar**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #130 
whatsapp image 2018-06-30 at 20 47 12

whatsapp image 2018-06-30 at 20 47 12 1

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
@anandkumarkparmar Please post a screenshot also of how the page looks after adding the option.

**Comment by anandkumarkparmar** [comment]
@Swati4star I have added 2 screenshots. Thanks

File: app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java [hunk 1]
```diff
@@ -163,6 +163,16 @@ public boolean onOptionsItemSelected(final MenuItem item) {
                             R.string.snackbar_no_images,
                             Snackbar.LENGTH_LONG).show();
                 }
+                break;
+            case R.id.item_share:
+                if (mViewFilesAdapter.areItemsForDeleteSelected()) {
+                    mViewFilesAdapter.shareFiles();
+                } else {
+                    Snackbar.make(Objects.requireNonNull(mActivity).findViewById(android.R.id.content),
```
[the remark this answers is not in the document]
Comment (anandkumarkparmar) [reply app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java]: Should I change for ```case R.id.item_delete```?

There also message is ```No Images selected```

[the remark this answers is not in the document]
Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java]: Yes! Thanks :)

**Pull request merged**